nautobot/dcim/models/cables.py

```diff
@@ -72,13 +72,13 @@ class Cable(PrimaryModel):
     )
     termination_b_id = models.UUIDField()
     termination_b = GenericForeignKey(ct_field="termination_b_type", fk_field="termination_b_id")
-    type = models.CharField(max_length=50, choices=CableTypeChoices, blank=True)
+    type = models.CharField(max_length=255, choices=CableTypeChoices, blank=True)
```

Comment:
As discussed in standup, let's revert `max_length` for any field with an explicit `choices` list.